docs/Contributing/Issues/issues.md

```diff
@@ -0,0 +1,70 @@
+## Using the issue tracker
+
+The [issue tracker](https://github.com/RI-SE/ATOS/issues) on the repo page is the preferred channel for [bug reports](#bugs) and features requests. If you want to submit a pull request that relates to a specific issue or bug, please select the bug in question and use the "create branch for this issue or link a pull request" option under the debelopment heading to link your pull request to the issue.
```

Comment:
```suggestion
The [issue tracker](https://github.com/RI-SE/ATOS/issues) on the repo page is the preferred channel for [bug reports](#bugs) and feature requests. If you want to submit a pull request that relates to a specific issue or bug, please select the bug in question and use the "create a branch for this issue or link a pull request" option under the development heading to link your pull request to the issue.
```

Reply:
If you want to submit a pull request that relates to a specific **issue or bug**, please select the **bug** in question